xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -81,9 +184,67 @@ ConfigUpdate build() {
    * configurations for traffic control such as drop overloads, inter-cluster load balancing
    * policy and etc.
    */
-  // TODO(zdapeng): content TBD.
   static final class EndpointUpdate {
+    private final String clusterName;
+    private final Map<Locality, LocalityLbEndpoints> localityLbEndpointsMap;
+    private final List<DropOverload> dropPolicies;
+
+    private EndpointUpdate(
+        String clusterName, Map<Locality,
+        LocalityLbEndpoints> localityLbEndpoints,
+        List<DropOverload> dropPolicies) {
+      this.clusterName = clusterName;
+      this.localityLbEndpointsMap = localityLbEndpoints;
+      this.dropPolicies = dropPolicies;
+    }
+
+    static Builder newBuilder() {
+      return new Builder();
+    }
 
+    String getClusterName() {
+      return clusterName;
+    }
+
+    /**
+     * Returns a map of localities with endpoints load balancing information in each locality.
+     */
+    Map<Locality, LocalityLbEndpoints> getLocalityLbEndpointsMap() {
+      return Collections.unmodifiableMap(localityLbEndpointsMap);
+    }
+
+    /**
+     * Returns a list of drop policies to be applied to outgoing requests.
+     */
+    List<DropOverload> getDropPolicies() {
+      return Collections.unmodifiableList(dropPolicies);
+    }
+
+    static final class Builder {
```

Comment:
The Builder can not be reused and `build()` for multiple times because its collections are highly mutable.

Reply:
Done. Make copies.